src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -190,12 +191,12 @@ bool emberAfGroupsClusterAddGroupCallback(EndpointId endpoint, app::CommandHandl
 
 bool emberAfGroupsClusterViewGroupCallback(EndpointId endpoint, app::CommandHandler * commandObj, GroupId groupId)
 {
-    EmberAfStatus status                                          = EMBER_ZCL_STATUS_NOT_FOUND;
-    CHIP_ERROR err                                                = CHIP_NO_ERROR;
-    uint8_t groupName[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH + 1] = { 0 };
+    EmberAfStatus status = EMBER_ZCL_STATUS_NOT_FOUND;
+    CHIP_ERROR err       = CHIP_NO_ERROR;
+    chip::ByteSpan groupName;
 
     // Get the group name, if supported
-    emberAfPluginGroupsServerGetGroupNameCallback(emberAfCurrentEndpoint(), groupId, groupName);
+    emberAfPluginGroupsServerGetGroupNameCallback(emberAfCurrentEndpoint(), groupId, &groupName);
```

Comment:
This also does not make sense.  What would make sense here would be passing in a buffer with `ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH` space in it and having the callee fill it in and adjust the size as needed.